packages/playground/src/weblets/tf_contracts_list.vue

```diff
@@ -178,16 +182,22 @@ const headers: VDataTableHeader = [
   { title: "Node Status", key: "nodeStatus", sortable: false },
   { title: "Details", key: "actions", sortable: false },
 ];
-
+const loadingError = ref("");
 async function onMount() {
+  loadingError.value = "";
   selectedContracts.value = [];
   loading.value = true;
   failedContractId.value = undefined;
   contracts.value = [];
```

Comment:
Can we reset these values in finally instead?

Reply:
we need to reset some values before starting, like hide the error message and remove field contracts ids, 
as we are not repeating this step inside catch, I think its fine to have them inside try block, what do you think ?

Reply:
some of them must be in try like error message; if we reset it in finally , we will never have an error message